app/views/components/gds/tariffLink.scala.html

```diff
@@ -0,0 +1,21 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+
+@(key: String)(implicit messages: Messages, appConfig: AppConfig)
```

Comment:
Instead of passing a key only, I'd pass the whole message. This way it is more generic.
In some places it turned out after some time that passing a key is limiting and either a new component needs to be built, or the old one is changed. In both cases there is some extra work to do, keeping in mind not to break existing things.